kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDataList.js

```diff
@@ -355,6 +359,7 @@ define([
         },
 
         refresh: function (showError) {
+            if(this.writtingLock) return;
```

Comment:
If (and other control) statements like this should be on a separate line.

We haven't talked about this yet, but we're mostly trying to follow the standard ESLint conventions -
 https://eslint.org/docs/developer-guide/code-conventions  They haven't been enforced much in the past, but now's the time! And we can clean up older code if we see issues. I'm pretty sure VSCode as ESLint built in, if not there's a plugin for it.

Reply:
Also, I didn't mark them, but there's a few instances where there's an if one-liner that should be surrounded by brackets. E.g.:
```
if (something) 
    doSomethingElse();
```
should be
```
if (something) {
    doSomethingElse();
}
```
I was reminded earlier that doing things like this can easily lead to dangerous bugs...

Reply:
In particular https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/